catppuccin.tmux

```diff
@@ -85,6 +85,15 @@ main() {
   set message-style "fg=${thm_cyan},bg=${message_background},align=centre"
   set message-command-style "fg=${thm_cyan},bg=${message_background},align=centre"
 
+  # menu
+  local menu_style menu_selected_style menu_border_style
+  menu_style=$(get_interpolated_tmux_option "@catppuccin_menu_style" "default")
+  menu_selected_style=$(get_interpolated_tmux_option "@catppuccin_menu_selected_style" "fg=${thm_black4},bg=${thm_yellow}")
```

Comment:
Why change to black4? I think gray was more readable.
![image](https://github.com/user-attachments/assets/bf6ae185-4fd8-4b29-a1af-608c6eb80911)
What do you think?

Reply:
just occurred to me I only tested 1 flavor.

Reply:
![image](https://github.com/user-attachments/assets/64e99f0d-cb11-426b-87b5-a277dedd28d3)
![image](https://github.com/user-attachments/assets/e7bd21a4-8f8c-4aff-b89b-c325b442c09c)
![image](https://github.com/user-attachments/assets/71ad51af-30e6-43e7-809f-fdcf25b83c13)
![image](https://github.com/user-attachments/assets/d7705ab5-c548-4f71-a679-c0ef4d48afc1)


gray looks in all better I think.

Reply:
I prefer gray too. I only changed to black4 because it seemed to match the main branch more closely.

Reply:
Gray it is!